src/main/java/subway/subway/Station.java

```diff
@@ -1,4 +1,4 @@
-package subway;
+package subway.subway;
```

Comment:
line과 같은 계층을 이룬다면 station이 더 잘 어울리지 않을까요?
```suggestion
package subway.station;
```

Reply:
앗 이부분 원래 목적이 station이었는데 잘못 지정해주었네요 변경하겠습니다!